<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AptOfflineQtInstallBugList</class>
 <widget class="QDialog" name="AptOfflineQtInstallBugList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>List of Bugs</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QListWidget" name="bugListViewWindow">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>630</y>
     <width>107</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close this window&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/application-exit.png</normaloff>:/icons/icons/application-exit.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>List of bugs</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="bugListplainTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
